dts/bindings/reset/reset-controller.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Andrei-Edward Popa <andrei.popa105@yahoo.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Reset Controller
+
+include: base.yaml
+
+properties:
+    reg:
+      required: true
+    reg-width:
+      type: int
+      required: false
+      description: The width of the reset registers in bytes. Default is 4 bytes.
+    active-low:
+      type: int
+      required: false
+      description: Set if reset is active low. Default is 0, which means active-high.
```

Comment:
This is not something related to the reset API. This should be moved to the reset controller yaml file.